app/mailers/user_mailer.rb

```diff
@@ -430,4 +436,8 @@ def email_should_receive_nonessential_notifications?(email)
     modified_email = email.gsub(/\+[^@]+@/, '@')
     !banlist.include?(modified_email)
   end
+
+  def deactivation_period_hours
```

Comment:
Since this class contains all mailers for a user and not specifically account reset, it might be a good idea to qualify this method name to specifically mention "account reset", so it's clear what it's used for.